<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>unirse_juego</class>
 <widget class="QWidget" name="unirse_juego">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>19</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="imagen">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-80</y>
     <width>871</width>
     <height>661</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="contraLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Contraseña</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="contraLineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Escriba la contraseña</string>
   </property>
  </widget>
  <widget class="QLabel" name="contraLabel_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nickname</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombreLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>9</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Escriba su nombre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonUnirse_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Unirse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonVolver">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>volver</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
